src/main/java/ch/naviqore/service/NotFoundException.java

```diff
@@ -0,0 +1,4 @@
+package ch.naviqore.service;
+
+public class NotFoundException extends Exception{
```

Comment:
Do we want to have checked exception? I think this could be a good way, or we just return nullable?

Reply:
I generally like custom exceptions, but I'm ok with all approaches. I now in web engineering I received an adverse reaction to having all kinds of custom exceptions, so I'm open to both